dev-test/githunt/types.reactApollo.customSuffix.tsx

```diff
@@ -355,9 +356,10 @@ export function useOnCommentAddedSubscription(
     OnCommentAddedSubscriptionVariables
   >
 ) {
+  const options = { ...defaultOptions, ...baseOptions };
```

Comment:
could we drop this when `defaultOptions` is empty (or `baseOptions` is not passed)? extra spreads for every single render is unnecessary and breaks any and all `useMemo` optimization

Reply:
This line  basically overrides default options from the Apollo client, so it would be great if the suggestion from @SimenB would be implemented

Reply:
Would you create a PR for this change?

Reply:
could someone move this into a new issue?